Dockerfile

```diff
@@ -1,19 +1,20 @@
 FROM golang:1.15.2-alpine
 
-MAINTAINER TFG Co <backend@tfgco.com>
+LABEL TFG Co <backend@tfgco.com>
+WORKDIR /go/src/github.com/topfreegames/khan
 
 EXPOSE 80
 
 RUN apk update
```

Comment:
A suggestion here: it's good to chain multiple installation commands in the same `RUN` to avoid caching issues. ([reference](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run))

Here, we can chain 3 `RUN` commands:

```Dockerfile
RUN apk update && \
         apk add git make g++ apache2-utils curl && \
         apk add --update bash
```